pkg/build/builder/sti.go

```diff
@@ -265,8 +267,8 @@ func (s *S2IBuilder) Build() error {
 		} else {
 			glog.V(2).Infof("No push secret provided")
 		}
-		glog.V(1).Infof("Pushing %s image ...", pushTag)
-		if err := pushImage(s.dockerClient, pushTag, pushAuthConfig); err != nil {
+		fmt.Fprintf(s.out, "Pushing %s image ...\n", pushTag)
```

Comment:
When I start with no loglevel set ('oc cluster up'), I get this (no v(1) messages):

```
...
Copying built war files into /wildfly/standalone/deployments for later deployment...
...done
Waiting to push 12 layers
Pushing 4 layers ( 10%), 1 pending
Pushing 5 layers ( 16%)
Pushing 4 layers ( 24%)
Pushing 4 layers ( 50%)
Pushing 3 layers ( 49%)
Pushing 3 layers ( 77%)
Pushing 2 layers ( 73%)
Pushing 1 layer  (100%)
```